docs/pages/reference/mesheryctl/subcommands/mesheryctl-sys-mesh-deploy.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: default
+title: mesheryctl system mesh deploy
+permalink: reference/mesheryctl/mesh/deploy
+type: reference
+display-title: "false"
+language: en
+# image: /assets/img/platforms/brew.png
+---
+
+<!-- Copy this template to create individual doc pages for each mesheryctl commands -->
+
+<!-- Name of the command -->
+# mesheryctl mesh deploy
+
+<!-- Description of the command. Preferably a paragraph -->
+## Description
```

Comment:
Description isn't visible on page. Add one

Reply:
Maybe you referenced the YAML in wrong manner. Figure it out to fix it

Reply:
I could be missing it - where is the description for this command?￼

Reply:
@alphaX86 could you verify it from here https://docs.google.com/spreadsheets/d/1q63sIGAuCnIeDs8PeM-0BAkNj8BBgPUXhLbe1Y-318o/edit#gid=0&range=B94? Thanks

Reply:
🙂👌